passiveData/src/commonMain/kotlin/org/sagebionetworks/assessmentmodel/passivedata/recorder/weather/WeatherServiceType.kt

```diff
@@ -0,0 +1,10 @@
+package org.sagebionetworks.assessmentmodel.passivedata.recorder.weather
+
+import kotlinx.serialization.Serializable
+import org.sagebionetworks.assessmentmodel.passivedata.internal.StringEnum
+
+@Serializable
+enum class WeatherServiceType : StringEnum {
+    Weather,
```

Comment:
Case-sensitive? ie. Shouldn't this have `@SerialName("weather")` and `@SerialName("airQuality")` because I defined them first using camelCasing standard for Swift enums?